src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/params.py

```diff
@@ -19,6 +19,11 @@
     c.register('condition', ('--condition',),
                type=json.loads,
                help='JSON encoded condition configuration. Use @{file} to load from a file.')
+    c.register('actions', ('--actions',),
+               type=json.loads,
```

Comment:
Raw references to `json.loads` should be changed to 'shell_safe_json_parse' from `azure.cli.core.util`